packages/shared/src/hooks/profile/useProfile.ts

```diff
@@ -13,8 +12,7 @@ export function useProfile(initialUser?: PublicProfile): PublicProfile {
     generateQueryKey(RequestKey.Profile, initialUser),
     () => getProfile(initialUser?.id),
     {
-      ...disabledRefetch,
-      cacheTime: StaleTime.OneHour,
+      staleTime: StaleTime.OneHour,
```

Comment:
This was probably meant to be `staleTime` since `cacheTime` does not really keep from refetching.

Reply:
Referencing PR that added it https://github.com/dailydotdev/apps/pull/3305